README.md

```diff
@@ -425,6 +425,19 @@ general we discourage its use for a number of reasons:
   which doesn't help.)
 
 
+## Using rspec-mocks with minitest 5.6+
+
+The latest versions of minitest also define an `expect` method. To use the
```

Comment:
I wrote it as `Minitest` in my first draft, but then changed if after checking `minitest`'s docs (https://github.com/seattlerb/minitest) and finding that they seems to use the lowercase `minitest` more? The exception seems to be when it's at the start of a sentence.